packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -223,6 +288,22 @@ export default declare<State>((api) => {
         path: NodePath<t.TaggedTemplateExpression> | NodePath<t.CallExpression>,
         state: State
       ) {
+        if (isJsx(path, state)) {
+          throw buildCodeFrameError(
+            `Found a \`jsx\` function call in the Babel output where one should not have been generated. Was Compiled was not set up correctly?
```

Comment:
```suggestion
            `Found a \`jsx\` function call in the Babel output where one should not have been generated. Was Compiled not set up correctly?
```

Reply:
oops ty for the catch, I probably should revise my english grammar 😆